src/elements/content-sidebar/hooks/useSidebarMetadataFetcher.ts

```diff
@@ -115,6 +116,30 @@ function useSidebarMetadataFetcher(
         [onApiError],
     );
 
+    const deleteMetadataInstanceSuccessCallback = (metadataInstance: MetadataTemplateInstance) => {
+        const updatedInstances = templateInstances.filter(
+            templateInstance => templateInstance.id !== metadataInstance.id,
+        );
+        console.log('updatedInstances', updatedInstances);
+        setTemplateInstances(updatedInstances);
+    };
+
+    const deleteMetadataInstance = React.useCallback(
+        (metadataInstance: MetadataTemplateInstance) => {
+            if (!file || !metadataInstance) {
+                return;
+            }
+
+            api.getMetadataAPI(false).deleteMetadata(
+                file,
+                metadataInstance,
+                deleteMetadataInstanceSuccessCallback,
+                onApiError,
+            );
```

Comment:
I noticed you added `isMetadataRedesign` to the `deleteMetadata` method. Shouldn't `true` be passed here, since we're actually using it in the redesign?